project/CodegenPlugin.scala

```diff
@@ -166,6 +213,10 @@ object CodegenPlugin extends AutoPlugin {
         version.value
       }
     },
+    dotnetVersion := {
+      val versionArray = version.value.split("-".toCharArray)
+      versionArray.head + "-rc" + versionArray.drop(1).head
+    },
```

Comment:
I added this for python because python and R dont allow fancy versions like ours. Is this also the case with .NET?